content/about/notes/1.0/index.md

```diff
@@ -3,18 +3,60 @@ title: Istio 1.0
 weight: 92
 ---
 
-This is a major release of 1.0. This version is a hardened version of 0.8 with a few new features in addition
-to usual pile of bug fixes and performance improvements.
+We're proud to release Istio 1.0! Istio has been in development for nearly two years, and the 1.0 release represents a substantial
+milestone for us. All of our [core features](/about/feature-stages/) are now stable and ready for production use.
 
-## Networking
+These release notes describe what's different between Istio 0.8 and Istio 1.0. Istio 1.0 only has a few new features
+relative to 0.8 as most of the effort for this release went into fixing bugs and improving performance.
+
```

Comment:
We should highlight istio ingress is deprecated and removed by default.  Also, the support for old traffic management rule (v1alpha1) is removed.

Reply:
Please document the changes in networking API resources: https://github.com/istio/api/compare/release-0.8...release-1.0#diff-0820bb0406265ba9b9a9e11dcec06601R666. I think it's mostly addition but there is a rename, which might prevent 0.8 config working with 1.0.

Reply:
@frankbu Can you write a bullet item for this please? Is there a page somewhere explaining this that we can point to?